app/coffee/BucketController.coffee

```diff
@@ -0,0 +1,36 @@
+PersistorManager = require("./PersistorManager")
+settings = require("settings-sharelatex")
+logger = require("logger-sharelatex")
+FileHandler = require("./FileHandler")
+metrics = require("metrics-sharelatex")
+parseRange = require('range-parser')
+Errors = require('./Errors')
+
+oneDayInSeconds = 60 * 60 * 24
+maxSizeInBytes = 1024 * 1024 * 1024 # 1GB
+
+module.exports = BucketController =
+
+	getFile: (req, res)->
+		{bucket} = req
```

Comment:
is bucket not on `req.params` rather than just `req`? Might well but but its new to me

Reply:
I was following the pattern of https://github.com/sharelatex/filestore-sharelatex/blob/master/app/coffee/FileController.coffee#L15

Reply:
I think bucket gets injected with `keyBuilder.userFileKey` https://github.com/sharelatex/filestore-sharelatex/blob/master/app.coffee#L70

Reply:
Sorry, I see these things and I just assume it's some magic in the package that I didn't know about before. Fixing.